core/src/mindustry/type/Planet.java

```diff
@@ -264,4 +264,12 @@ public void displayInfo(Table table){
     public ContentType getContentType(){
         return ContentType.planet;
     }
+
+    public boolean visible(){
```

Comment:
Why not make it as class property?

Reply:
what if i want it to be visible only when its day irl, or when some block is researched, etc.